HW_7_Task_2/HW_7_Task_2/Clock.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace HW_7_Task_2
+{
+    public partial class Clock : Form
+    {
+        public Clock()
+        {
+            InitializeComponent();
+        }
+
+        private Graphics graphics;
+        private Timer timer = new Timer();
```

Comment:
```suggestion
        private Timer timer = new();
```
Не надо писать тип дважды